lib/services/api/messages.ts

```diff
@@ -14,8 +14,8 @@ export const API_MESSAGE_FALLBACK_LOGIN_FAIL       = 'Invalid email or password.
 export const API_MESSAGE_GENERIC_TOKEN_REFRESH_FAIL= `A server error occurred while trying to refresh your session.
                                                       Please try again.`;
 
-export const API_MESSAGE_TOKEN_REFRESH_IN_PROGRESS = `Your account session is being renewed with the server.
-                                                      Please try your request again.`;
+export const API_MESSAGE_TOKEN_REFRESH_IN_PROGRESS = `Your account session is being renewed with the server. \
```

Comment:
@mobitar I had to modify this string a bit. I wanted to test the message in https://github.com/standardnotes/snjs/commit/824c6d6f90b9dba5072ce86598f4f56bcf739627#diff-3625700b711b80a59b096f9544c87c27R262 but it wouldn't let me do so (unless that I copied the exact string with spaces, which doesn't sound to be a good approach). 

Any comments on this matter? Should we use this string format for the rest of messages?

Reply:
We don't actually want newlines in these messages. We should instead use "" + "" syntax for multi-line strings. It was a mistaken assumption on my end that using `` syntax would automatically ignore newlines in the JS files.